content/publications.md

```diff
@@ -4,31 +4,34 @@ date: 2022-08-12T09:46:31+02:00
 
 # Publications
-* ITEA4 Eureka Cluster on Software Innovation: "ITEA Success story APPSTACLE Vehicle connectivity for novel applications", 2022 https://itea4.org/project/success-story/appstacle-succes-story-1.html
-* D. Grewe, M. Wagner, U. Ambalavanan, L. Liu, N. Nayak and S. Schildt: "On the Design of an Information-Centric Networking Extension for IoT APIs",  2021 IEEE 94th Vehicular Technology Conference (VTC2021-Fall), 2021, https://ieeexplore.ieee.org/abstract/document/9625470
-* Junhyung Ki, Sebastian Schildt, Andreas Hastall Sven Erik Jeroschewski, Robert Hottger: "Eclipse KUKSA.val for SCR Anti-Tampering Monitoring in Heavy Vehicles", 2nd Eclipse Research International Conference on Security, Artificial Intelligence, Architecture and Modelling for Next Generation Mobility, 2021, http://ceur-ws.org/Vol-3028/D2-02-ESAAMM_2021_paper_3.pdf 
-* A. Banijamali A., P. Kuvaja, M. Oivo, P. Jamshidi:
-"Kuksa*: Self-adaptive Microservices in Automotive Systems",
- Lecture Notes in Computer Science (including subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics), 2020, https://link.springer.com/chapter/10.1007/978-3-030-64148-1_23
-* Hirvonsalo, Harri & Seppänen, Pertti (2021) On deployment of Eclipse Kuksa as a framework for an intelligent moving test platform for research of autonomous vehicles. In: Pasi Kuvaja et al. (eds.) CEUR workshop proceedings Vol. 3028, Proceedings of the 2nd Eclipse Research International Conference on Security, Artificial Intelligence, Architecture and Modelling for Next Generation Mobility (pp. 38-50), http://ceur-ws.org/Vol-3028/D2-03-ESAAMM_2021_paper_8.pdf
-* P. Heisig, S. E. Jeroschewski, J. Kristan, R. Höttger, A. Banijamali, S. Sachweh: "Bridging the Gap between SUMO & Kuksa: Using A Traffic Simulator for Testing Cloud-based Connected Vehicle Services", SUMO User Conference 2019, https://easychair.org/publications/paper/4Zsv  
-* M. Wagner, S.Schildt, Innovation durch Offenheit: Das Open Source Connected Vehicle Framework Eclipse Kuksa, Boardnetz Kongress 2019, https://www.researchgate.net/profile/Marco-Wagner/publication/330281127_Innovation_through_Openness_-_The_Open_Source_Connected_Vehicle_Framework_Eclipse_Kuksa/links/5c371bcda6fdccd6b5a0a357/Innovation-through-Openness-The-Open-Source-Connected-Vehicle-Framework-Eclipse-Kuksa.pdf
-* Banijamali A., Jamshidi P., Kuvaja P., Oivo M.:
-"Kuksa: A Cloud-Native Architecture for Enabling Continuous Delivery in the Automotive Domain",
-Lecture Notes in Computer Science (including subseries Lecture Notes in Artificial Intelligence and Lecture Notes in Bioinformatics), 2019,
-https://link.springer.com/chapter/10.1007/978-3-030-35333-9_32
-* Höttger R., Özcelikörs M., Heisig P., Krawczyk L., Cuadra P., Wolff C.: "Combining eclipse IoT technologies for a RPI3-Rover along with Eclipse Kuksa",
-CEUR Workshop Proceedings 2018, http://ceur-ws.org/Vol-2066/seerts2018paper03.pdf
\ No newline at end of file
+
```

Comment:
Any changes here? Just curious github sees a diff :)

Reply:
code formatter added this line :)